ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -805,6 +806,16 @@ objects:
             exit 1
         fi
 
+        # as a current stop gap -- this is pointing to a proxy hosted in
+        # the namespace "ci-test-ewolinet" on the ci cluster
+        if [[ "${ENABLE_PROXY}" == "true" ]]; then
+          cat >> /tmp/artifacts/installer/install-config.yaml << EOF
+        proxy:
+          httpsProxy: http://admin:admin@35.196.128.173:3128/
+          httpProxy: http://admin:admin@35.196.128.173:3128/
```

Comment:
@ewolinetz is `noProxy` not required here?

Reply:
the installer injects some standard values, see [here][1] and surrounding discussion.

[1]: https://github.com/openshift/release/pull/4591#discussion_r311191558